client/src/component/Dashboard/Dashboards/Company/views/CompanyProfile.jsx

```diff
@@ -1,15 +1,41 @@
-import React, { Component} from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios'
 
-class CompanyProfile extends Component {
-  state = {
-    
-  }
+const CompanyProfile = () => {
+  const [companyInfo, setCompanyInfo] = useState({
+    id: "1",
```

Comment:
another hardcoded value. Will be interesting to figure out how I can send this component the company id from the job post component since currently, this is setup to read from the company user id who is currenlty logged in. I wonder how we can display this information for a bootcamp user who's currenlty logged in.